component/libfuse/libfuse2_handler.go

```diff
@@ -428,6 +428,13 @@ func libfuse2_readdir(_ *C.char, buf unsafe.Pointer, filler C.fuse_fill_dir_t, o
 			}
 		}
 
+		if off_64 == 0 {
+			flags := internal.NewDirBitMap()
```

Comment:
these flags are static in nature, we can define them as class variables and use the same all the time instead of creating on each readdir.

Reply:
just updated